e2e-tests/batch-mode-tests.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+print_prefix="E2E: BATCH MODE TEST:"
+echo "START OF BATCH MODE TEST"
+echo "${print_prefix} STARTING OPENMRS SERVER"
+docker-compose -f openmrs-compose.yaml up -d
+statuscode=$(curl -o /dev/null --head -w "%{http_code}" -L -X GET -u admin:Admin123 http://localhost:8099/openmrs/ws/rest/v1/visittype)
+until [[ ${statuscode} -eq 200 ]]; do
+      sleep 40s;
+      echo "${print_prefix} WAITING FOR OPENMRS SERVER TO START"
+      statuscode=$(curl -o /dev/null --head -w "%{http_code}" -L -X GET -u admin:Admin123 http://localhost:8099/openmrs/ws/rest/v1/visittype)
+done
+echo "${print_prefix} OPENMRS SERVER STARTED SUCCESSFULLY"
+echo "${print_prefix} BUILD THE ANALYTICS PROJECT"
+mvn clean install -DskipTests=true 
+rm -rf /tmp/analytics-tests
+mktemp -d /tmp/analytics-tests
+echo "${print_prefix} RUN BATCH MODE WITH PARQUET SYNC"
+java -cp batch/target/fhir-batch-etl-bundled-0.1.0-SNAPSHOT.jar org.openmrs.analytics.FhirEtl --openmrsServerUrl=http://localhost:8099/openmrs --openmrsUserName=admin --openmrsPassword=Admin123  --searchList=Patient,Encounter,Observation --batchSize=20 --targetParallelism=20 --fileParquetPath=/tmp/analytics-tests/
+if [[ -z $(ls /tmp/analytics-tests/Patient) ]]; then
+  echo "${print_prefix} DATA NOT SYNCED TO PARQUET FILES"
+  exit 0
+fi
+echo "${print_prefix} PARQUET FILES SYNCED IN /tmp/analytics-tests DIRECTORY"​
+current_path=`pwd`
+echo "${print_prefix} COPYING PARQUET TOOLS JAR FILE TO ROOT"
+cp e2e-tests/parquet-tools-1.11.1.jar /tmp/analytics-tests/
+if ! [[ $(command -v jq) ]]; then
+    echo "INSTALLING JSON COMMAND-LINE PROCESSOR, JQ" >&2
+    if [[ "$OSTYPE" == "linux-gnu"* ]]; then
+        echo "OSTYPE: linux"
+        sudo apt install jq
+    elif [[ "$OSTYPE" == "darwin"* ]]; then
+        echo "OSTYPE: Mac OSX"
+        brew install jq
+    fi
+fi
+
+if ! [[ $(command -v jq) ]]; then
+  echo "COULD NOT INSTALL JQ, INSTALL MANUALLY TO CONTINUE RUNNING THE TEST" >&2
+  brew install jq
+fi
+cd /tmp/analytics-tests
+mkdir omrs
+echo "${print_prefix} Finding number of patients, encounters and obs in openmrs server"
+curl -L -X GET -u admin:Admin123 http://localhost:8099/openmrs/ws/fhir2/R4/Patient/ >> ./omrs/patients.json
```

Comment:
This (potentially) allows us to do less work on the backend

```suggestion
curl -L -X GET -u admin:Admin123 http://localhost:8099/openmrs/ws/fhir2/R4/Patient/?_summary=count >> ./omrs/patients.json
```